packages/mobile/src/onboarding/UseBackToWelcomeScreen.tsx

```diff
@@ -59,6 +60,7 @@ export function useBackToWelcomeScreen({ backAnalyticsEvents }: Props) {
           ValoraAnalytics.track(analyticsEvent)
         })
         dispatch(cancelCreateOrRestoreAccount())
+        dispatch(cancelRedeemInvite())
```

Comment:
What is the difference between this event and the once above it?

Reply:
The new action switches `isRedeemingInvite` to `false`. When a user was hitting cancel, this could be remaining as `true`, which would prevent the user from being able to skip to the next screen.

Reply:
In general, using consecutive `dispatch` should be avoided for 2 reasons:
- I think it's important to have meaningful actions and it's good to have such actions cause different parts of the state to change and cause multiple side effects.
- We don't want incorrect state to be possible. Dispatching 2 actions consecutively makes it possible to end up with incorrect state at least temporarily. And definitely if there's a crash/app being killed between the two, though quite unlikely of course.

Conceptually we're still creating the account here. And it looks normal to have the `cancelCreateOrRestoreAccount` cancel/reset any state linked to it.

I propose we remove the newly introduced `cancelRedeemInvite` action and react to `cancelCreateOrRestoreAccount` in the redeem invite saga.

Reply:
Good suggestion. Will do.